src/ucp/core/ucp_ep.h

```diff
@@ -12,7 +12,6 @@
 
 #include <uct/api/uct.h>
 #include <ucs/debug/log.h>
-#include <ucs/debug/log.h>
```

Comment:
@shamisp Sorry for this. I thought it is minor change in the same file I worked on. Creating separate PR for every small change increase number of PRs and review\merge work.
I will do the way you proposed if you like.